pkg/agent/secondarynetwork/podwatch/controller.go

```diff
@@ -134,6 +135,15 @@ func NewPodController(
 		},
 		resyncPeriod,
 	)
+
+	if err := pc.initializeSecondaryInterfaceStore(); err != nil {
```

Comment:
Probably let us check ovsBridgeClient != nil here, and otherwise skip initializeSecondaryInterfaceStore and reconcileSecondaryInterfaces. Please add a comment explaining this happen when secondary bridge is not configured.